src/main/java/ladder/model/Participants.java

```diff
@@ -0,0 +1,32 @@
+package ladder.model;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Participants {
+
+    private final List<Name> names;
+
+    private Participants(List<Name> names) {
+        this.names = names;
+    }
+
+    public static Participants from(String input) {
+        return new Participants(
+            Arrays.stream(input.split(","))
+                  .map(String::trim)
+                  .map(Name::new)
+                  .collect(Collectors.toList())
+        );
+    }
+
+    public int size() {
+        return names.size();
+    }
+
+    public List<Name> values() {
+        return new ArrayList<>(names);
+    }
```

Comment:
`Participants` 클래스는 게임의 참여자를 의미하는 클래스 같네요!

그리고 `values` 메서드는 참여자의 이름을 모두 반환하도록 구현되어 있네요.

하지만 `values` 라는 이름을 보면, `참여자의 이름 목록`이라는 의미가 명확하게 전달되지 않아 보여요.

Reply:
public List<Name> getParticipantsNameList() {
}

으로 메서드명을 좀 더 명확하게 수정하였습니다!

Reply:
```suggestion
    public List<Name> getParticipantsNameList() {
        return new ArrayList<>(names);
    }
```
 메서드명을 좀 더 명확하게 수정하였습니다!

Reply:
추가로, List 타입(더 정확히는 Collection 인터페이스를 구현한)의 변수명 또는 메서드명의 접미사에는 `List`와 같은 이름을 붙이는 것보다는 복수형을 붙이는 것이 더 권장돼요.

이유는 중복된 의미가 부여되기 때문이고(타입이 이미 List인데 굳이 List라는 이름을 한 번 더 사용), 추후 List 타입이 아닌 다른 타입(Set)으로 변경되었을 때, 변수명까지 한 번에 변경해야 하기 때문이에요.

하지만 전 지금까지 기존에 선언된 List 타입의 변수를 다른 타입으로 변경한 적은 없어요.
하지만 매개변수의 타입은 유연하게 받는 경우가 있어, 이 점은 참조만 해주시면 될 것 같아요.
```java
List<User> findByIdIn(Collection<Long> ids);